packages/client/src/Home/Home.jsx

```diff
@@ -13,11 +13,12 @@ function hashFromTuples(tuples) {
   }, {});
 }
 
-async function sendRequest({ endpoint, method, qs }) {
+async function sendRequest({endpoint, method, qs, version}) {
   const response = await httpClient[method]({
-    base: `/${endpoint}`,
+    base: `${endpoint}`,
     qs: hashFromTuples(qs),
     params: {},
+    api_version: version,
```

Comment:
All the magic happens here :)

Reply:
Since we are using only `EndpointConfig` parameter type and not using `string` parameter type at all, currently we are not supporting "full resource path" eg: https://app.procore.com/check_company_id